packages/sdk/package.json

```diff
@@ -17,6 +17,7 @@
     "lint:fix": "yarn lint:check --fix",
     "pre-commit": "lint-staged",
     "test": "hardhat test",
+    "test:next": "vitest test-next/proveMessage.spec.ts",
```

Comment:
Are you planning on moving everything away from `hardhat` for testing?

Reply:
ah ok i see your comment

Reply:
yes but would solicit feedback first